bears/general/FilenameBear.py

```diff
@@ -19,14 +20,20 @@ class FilenameBear(LocalBear):
                           'snake': to_snakecase,
                           'space': to_spacecase}
 
+    _language_naming_convention = {'.java': 'pascal',
+                                   '.js': 'kebab',
+                                   '.py': 'snake'}
```

Comment:
This should be a module level private variable, and it will be extended, so the layout should be friendly to additions.
```py
x = {
  a,
  b,
}
```